recipe/activate.sh

```diff
@@ -6,34 +6,70 @@
 
 # doesn't come with a deactivate script, because the symlink
 # is benign and doesn't need to be deleted.
+CF_LIBARROW_ACTIVATE_LOGGING=1
+_la_log() {
+    if [ "$CF_LIBARROW_ACTIVATE_LOGGING" = "1" ]; then
+        # The following loop is necessary to handle multi-line strings
+        # like for the output of `ls -al`.
+        while IFS= read -r line; do
+            echo "${BASH_SOURCE[0]} DEBUG: $line"
```

Comment:
While the script implies bash at the top, I think this is also executed with other shells like `zsh` and thus this will break them.

Reply:
Oh yikes, thanks for pointing this out.

Would this solve it?

```diff
     if [ "$CF_LIBARROW_ACTIVATE_LOGGING" = "1" ]; then
         # The following loop is necessary to handle multi-line strings
         # like for the output of `ls -al`.
-        while IFS= read -r line; do
-            echo "${BASH_SOURCE[0]} DEBUG: $line"
-        done <<< "$*"
+        printf '%s\n' "$*" | while IFS= read -r line
+        do
+            echo "$CONDA_PREFIX/etc/conda/activate.d/libarrow_activate.sh DEBUG: $line"
+        done
     fi
 }
```

Reply:
Yes, this will work.